### ISSUE TITLE
'pause', 'resume', 'start', 'stop' not working

### ISSUE THREAD

**Issue by guyht** [hilios/jQuery.countdown#22]
According to the documentation, the following code should setup up a timer, then stop it, but instead I get the following error:

    [Error] Error: Couldn't cast `stop` to a date object.
	parseDateString (jquery.countdown.js, line 63)
	setFinalDate (jquery.countdown.js, line 181)
	Countdown (jquery.countdown.js, line 140)
	(anonymous function) (jquery.countdown.js, line 245)
	each (jquery.js, line 4)
	each (jquery.js, line 4)
	countdown (jquery.countdown.js, line 225)
	(anonymous function) (t.html, line 13)
	l (jquery.js, line 4)
	fireWith (jquery.js, line 4)
	ready (jquery.js, line 4)
	S (jquery.js, line 4)

Test code:

    <html>
    <head>
        <script src="public/js/jquery.js"></script>
        <script src="public/js/jquery.countdown.js"></script>
    </head>
    <body>
        <div id="count"></div>
        <script>
            $(function() {
                $('#count').countdown('10/10/2014', function(event) {
                    $(this).html(event.strftime('%H:%M:%S'));
                });
                $('#count').countdown('stop');
                   });
        </script>

**Comment by reidlai** [comment]
I am using 2.0.4.  When I try to reset JQuery Countdown value and restart by using start method, I encounter the error "Couldn't cast `start` to a date object".  Is it replicated problem?  Thanks for your advice.

**Comment by birdkiwi** [comment]
I have this problem too. 2.0.4

**Comment by hilios** [comment]
:+1:

**Comment by hilios** [comment]
Pl

**Comment by hilios** [comment]
I couldn't reproduce the issue reported, I uploaded a demo page at the docs to test this functionality and everything seems OK

http://hilios.github.io/jQuery.countdown/examples/bootstrap.html

**Comment by iancampelo** [comment]
Hi Hilios, 

I'm having an issue... When I've paused and after that I Resume, the countdown don't start where it's stopped. How can I fix that?
http://hilios.github.io/jQuery.countdown/examples/bootstrap.html
Thanks

**Comment by hilios** [comment]
@iancampelo read this topic:
https://github.com/hilios/jQuery.countdown/issues/144#issuecomment-146582246

**Comment by tctimmeh** [comment]
We just ran into this issue with version 2.2.0. In our case, the problem was that we were setting up the timer _before_ the target element was added to the page. It seems that the timer gets automatically [removed if the element isn't on the page yet](https://github.com/hilios/jQuery.countdown/blob/2.2.0/src/countdown.js#L206), even when it's first created.

We worked around the problem by wrapping the timer setup code in a `setTimeout()` call.
